src/Process/3dTilesProcessing.js

```diff
@@ -79,7 +79,51 @@ export function $3dTilesCulling(node, camera) {
     return false;
 }
 
+// Cleanup all 3dtiles|three.js starting from a given node n.
+// n's children can be of 2 types:
+//   - have a 'content' attribute -> it's a tileset and must
+//     be cleaned with cleanup3dTileset()
+//   - doesn't have 'content' -> it's a raw Object3D object,
+//     and must be cleaned with _cleanupObject3D()
+function cleanup3dTileset(layer, n) {
+    for (let i = 0; i < n.children.length; i++) {
+        // skip non-tiles elements
+        if (!n.children[i].content) {
+            n.children[i].traverse(_cleanupObject3D);
+        } else {
+            cleanup3dTileset(layer, n.children[i]);
+        }
+    }
+    if (n.dispose) {
+        n.dispose();
+    }
+    delete n.content;
+    layer.tileIndex.index[n.tileId].loaded = false;
+    n.remove(...n.children);
+}
```

Comment:
Don't you want to call `cleanupObject3D` on n too'?

Reply:
No, tiles doesn't have a geometry + material. They're only a container for mesh (stored in children and content) and other tiles (in children).